src/main/java/com/prgms/voucher/voucherproject/domain/PercentDiscountVoucher.java

```diff
@@ -0,0 +1,31 @@
+package com.prgms.voucher.voucherproject.domain;
+
+import com.prgms.voucher.voucherproject.domain.Voucher;
+
+import java.util.UUID;
+
+public class PercentDiscountVoucher implements Voucher {
+    private final UUID voucherId;
+    private final long percent;
+
+
+    public PercentDiscountVoucher(UUID voucherId, long percent) {
+        this.voucherId = voucherId;
+        this.percent = percent;
+    }
+
+    @Override
+    public UUID getVoucherId() {
+        return null;
+    }
+
+    @Override
+    public long discount(long beforeDiscount) {
+        return beforeDiscount * (percent/100);
```

Comment:
산술 연산자 전후로 띄어쓰기가 안되어 있어요. 확인 부탁 드릴게요.

```suggestion
        return beforeDiscount * (percent / 100);
```